### PR TITLE
CommonServerPython: add handler for py warnings

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Handle python warnings. Currently if a library issues a warning it goes to stderr which will create an error entry (see test PB). This PR changes so we log it to `demisto.info`.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by glicht** [comment]
TDD: failing PB test before the fix: https://app.circleci.com/pipelines/github/demisto/content/56917/workflows/5ca24fa8-4369-4b16-9964-3f77c3182d5f

**Review by Itay4 — APPROVED** [review]

File: Packs/DeveloperTools/TestPlaybooks/script-TestPyCommonServer.yml [hunk 1]
```diff
@@ -470,6 +470,11 @@ script: >-
       appendContext('zero_key', 0)
       appendContext('none_key', None)
       demisto.results("placeholder so the run would finish")
+  
+  def test_warnings():
```
Comment (yaakovi) [inline Packs/DeveloperTools/TestPlaybooks/script-TestPyCommonServer.yml:474]: nice test

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -21,6 +21,32 @@
...
+
+    def __init__(self):
+        self.org_handler = warnings.showwarning
+        warnings.showwarning = WarningsHandler.handle_warning
+
+    def __del__(self):
+        warnings.showwarning = self.org_handler
+
+
+_warnings_handler = WarningsHandler()
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:47]: where is `_warnings_handler` being used?

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -4878,8 +4904,7 @@ class DebugLogger(object):
         Is used when `debug-mode=True`.
     """
 
-    def __init__(self):
-        logging.raiseExceptions = False
+    def __init__(self):
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4907]: to fix the build
```suggestion
    def __init__(self):
```

**Review by yaakovi — APPROVED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 2 above)
Comment (glicht) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:47]: It is just an object ref so the __del__ function is called only when the script fully ends.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 3 above)
Comment (glicht) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py]: thanks. fixed already.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.yml [hunk 4]
```diff
@@ -16,6 +16,7 @@ timeout: 0s
 alt_dockerimages: # used for unit testing
 - demisto/python3:3.7.3.286
 - demisto/python3:3.8.3.8494
+- demisto/python3:3.9.1.14969
```
Comment (glicht) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.yml:19]: @anara123 @yaakovi @Itay4 note that I also added to run unit testing on py 3.9.1

**Pull request merged**